.github/workflows/build-test.yml

```diff
@@ -0,0 +1,54 @@
+name: 'Build and test'
+
+on:
+  workflow_call:
+  workflow_dispatch:
+  # not running this on main as it's pulled in by the deploy workflow anyway
+  # push:
+  #   branches:
+  #     - main
+  pull_request:
+
+permissions:
+  contents: read
+
+# This allows a subsequently queued workflow run to interrupt previous runs
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.13.x
+          cache: npm
+      - run: pipx install poetry
+      - uses: actions/setup-python@v4
+        with:
+          cache: 'poetry'
+
+      - name: Install Global JS dependencies
+        run: npm install -g ajv-cli@5.0.0 ajv-formats@2.1.1
+
+      - name: Install JS
+        run: npm ci
+
+      - name: Install Python
+        run: poetry install
+
+      - name: Build
+        run: scripts/build
+
+      - name: Run all tests
+        run: scripts/test
+
+      - name: Upload HTML
+        uses: actions/upload-pages-artifact@v1.0.6
+        with:
+          path: site
```

Comment:
Does this path match the output of the build script? I could't tell easily.

Reply:
Ah, yes it does, the line that does it is this https://github.com/alphagov/di-identity-vocab/blob/87ceeab90765256c54e94a8b16e9c698da5fb7f1/scripts/build#L16 and the default is "site" per https://www.mkdocs.org/user-guide/configuration/#site_dir